src/Public.Api/Infrastructure/Startup.cs

```diff
@@ -105,6 +102,20 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 ? baseUrl.Substring(0, baseUrl.Length - 1)
                 : baseUrl;
 
+            var amazonDynamoDbClient = new AmazonDynamoDBClient(RegionEndpoint.EUWest1);
+            if (_configuration["ASPNETCORE_ENVIRONMENT"] == "Development")
```

Comment:
Can't you check this using the IWebHost'ish interface with the `IsDevelopment()` method?